internal/storage/postsql/read.go

```diff
@@ -882,6 +882,10 @@ func addInstanceFilters(stmt *dbr.SelectStmt, filter dbmodel.InstanceFilter) {
 			stmt.Where("instances.deleted_at = '0001-01-01T00:00:00.000Z'")
 		}
 	}
+
+	if filter.BindingExists != nil && *filter.BindingExists {
+		stmt.Where("instances.instance_id IN (SELECT instance_id FROM bindings)")
```

Comment:
For each instance we scan entire bindings table since there is no `where` clause in subquery.

Reply:
```
broker=> explain select * from instances where instance_id in (select instance_id from bindings);
                                 QUERY PLAN                                  
-----------------------------------------------------------------------------
 Hash Semi Join  (cost=11.35..130.34 rows=60 width=1019)
   Hash Cond: ((instances.instance_id)::text = (bindings.instance_id)::text)
   ->  Seq Scan on instances  (cost=0.00..118.05 rows=105 width=1019)
   ->  Hash  (cost=10.60..10.60 rows=60 width=516)
         ->  Seq Scan on bindings  (cost=0.00..10.60 rows=60 width=516)
(5 rows)
```

Reply:
```sql
select * from instances where exists (select instance_id from bindings where bindings.instance_id=instances.instance_id);
```
This could be more optimal (at least not worse).